matrix-meetings-widget/src/components/meetings/MemberSelectionDropdown/MemberSelectionDropdown.tsx

```diff
@@ -246,22 +241,16 @@ export function MemberSelectionDropdown({
             'aria-describedby': instructionId,
             endAdornment: (
               <>
-                {showSelectAll && (
-                  <InputAdornment
-                    position="end"
-                    sx={{
-                      right: 60,
-                      top: 'calc(50%)',
-                      position: 'absolute',
-                    }}
-                  >
-                    <Button
+                {loading && (
+                  <InputAdornment position="end">
+                    <CircularProgress
+                      aria-label={t(
+                        'memberSelectionDropdown.loadingUsers',
+                        'Loading users…'
+                      )}
```

Comment:
We usually try to avoid defining translations in more than one place. So if a translated string is used in two places (here and in the `loadingText` prop), it's best to create a variable like `const loadingText = t(...);`. Otherwise it is easy to introduce bugs (or at least inconveniences) when changing the texts in the future.